ui/src/app/components/webcomponents/action-step-form.component.ts

```diff
@@ -1856,4 +1856,7 @@ export class ActionStepFormComponent extends BaseComponent implements OnInit {
       this.showTestDataPopup();
     }
   }
+  argumentsKeyRefactor(key:string){
```

Comment:
can we move this to global utils?
this can be used by other modules @manikanta-reddy-t 
ex, utils/string.ts can export this func where the func name can be like extractStringByKey where key can be optional